src/signals/my-incidents/components/IncidentsDetail/IncidentsDetail.tsx

```diff
@@ -28,6 +32,14 @@ export const IncidentsDetail = ({
 }: Props) => {
   const { _display, text, location, extra_properties, _links } = incidentsDetail
   const attachments = _links?.['sia:attachments']
+  const [showAttachmentViewer, setShowAttachmentViewer] = useState('')
```

Comment:
showAttachmentViewer indicates a boolean that makes the modal appear or disappear. I think you mean something like `selectedAttachement`. And i'd make the default `null`

Reply:
Done and replaced it. Checked with the other incidentDetail and there it is a boolean.